src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -82,6 +82,7 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Microsoft.VisualStudio.OLE.Interop" />
```

Comment:
why doesn't this have the same definition as the one in MIDebugEngine? either stick to not having the version and public key or include it for both

Reply:
I vote for the former.